FreeTubeApp/FreeTube
# Update History & UserPlaylists view to filter with delay

## Pull Request Type


## Related issue



N/A

## Description
When using filtering on history view
I found it laggy due to list being filtered on every char input
This PR makes the filtering happens 1s after last input (except when query is clear) (feel free to suggest other values, UX changes...)

## Screenshots 

https://user-images.githubusercontent.com/1018543/233236060-72c89582-96b2-4336-ace2-da3e12d595a2.mov


## Testing 



- Browse History/UserPlaylist
- Type something to filter
- Wait a bit and ensure filtering is done

## Desktop

- **OS:**
- **OS Version:**
- **FreeTube version:**

## Additional context
